wurst/_handles/Player.wurst

```diff
@@ -5,9 +5,11 @@ import Unit
 
 /** Use this array instead of Player() to avoid leaks */
 public player array players
+public player localPlayer
 init
 	for i = 0 to bj_MAX_PLAYER_SLOTS-1
 		players[i] = Player(i)
+	localPlayer = GetLocalPlayer()
```

Comment:
No need to have this in `init` block, I don't think?

If it is needed: needs a comment

Reply:
Will inline into a constant